recipes/gcc/all/conanfile.py

```diff
@@ -0,0 +1,128 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from conans.errors import ConanException, ConanInvalidConfiguration
+import os
+
+
+class GccConan(ConanFile):
+    name = "gcc"
+    description = "The GNU Compiler Collection includes front ends for C, " \
+                  "C++, Objective-C, Fortran, Ada, Go, and D, as well as " \
+                  "libraries for these languages (libstdc++,...). "
+    topics = ("gcc", "gnu", "compiler", "c", "c++")
+    homepage = "https://gcc.gnu.org/"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "	GPL-3.0-only"
```

Comment:
I have seen this a lot: spaces in front of the license ID. How is this happening? Is this a copy&paste thing from the SPDX list which contains somehow spaces or something similar?

Reply:
this is mostly human error, I've just copy-pasted from the SPDX web-site